starkyModules/metadataRoles.ts

```diff
@@ -0,0 +1,83 @@
+import { retrieveAssets } from "../aspect/retrieveAssets";
+
+import { StarkyModuleConfig, StarkyModuleField } from "./types";
+
+export const name = "Metedata Roles";
+
+export const fields: StarkyModuleField[] = [
+  {
+    id: "contractAddress",
+    question: "What's the ERC-721 contract address?",
+  },
+  {
+    id: "conditionPattern",
+    question: "What's the path & pattern to get the role?",
```

Comment:
I think this one is too complicated to understand without a placeholder.

Let's precise Regex for people that know what that is
```suggestion
    question: "What's the path & pattern (regex) to get the role?",
```

Also I think we should give an example. Can you add an optional parameter `placeholder` to `StarkyModuleField` and fill in the textarea/input field with it
You can call `setPlaceholder` on the `TextInputBuilder` in `addConfigCommand.ts` on line ~252

Reply:
The length of the question string should be <= 45, is there a reason why it's limited to 45 ?

Reply:
![Capture d’écran 2022-12-15 à 17 50 53](https://user-images.githubusercontent.com/2102342/207920029-b3bbb7fa-4577-45c8-ba2b-35733857102e.png)
Apparently it's a Discord limitation. Placeholder max length is 100. Let's not enforce it in code right now (no easy typescript type to say a string must have a given length and it will fail when devs try to put a longer string anyway)